packages/dds/merge-tree/src/test/tsconfig.json

```diff
@@ -10,7 +10,7 @@
         ],
     },
     "include": [
-        "./**/*"
+        "./**/*", "../text.ts"
```

Comment:
why is this needed?

Reply:
ummm, I was moving things around and I think vscode added it... I will remove.